wordpress-mobile/WordPress-iOS
# Add passkeys support back

To test:

- Verify that passkeys login works (see https://github.com/wordpress-mobile/WordPress-iOS/pull/22001)
- Verify that passkeys from third-party password managers, e.g. 1Password, work
- Verify that universal links for *.wordpress.com subdomains work
- (Regression Testing) Verify that Login with Apple works
- (Regression Testing) Verify that password autofill works

> [!IMPORTANT]  
> On my physical device, it required a device restart to start working

## Regression Notes
1. Potential unintended areas of impact: Passkeys, Universal Links
2. What I did to test those areas of impact (or what existing automated tests I relied on): manual
3. What automated tests I added (or what prevented me from doing so): n/a

PR submission checklist:


UI Changes testing checklist:
